crates/holochain/src/conductor/conductor.rs

```diff
@@ -515,14 +515,30 @@ where
         }
     }
 
-    /// Create Cells for each CellId marked active in the ConductorState db
-    pub(super) async fn create_active_app_cells(
+    /// Remove Cells from the conductor for each CellId marked stopped. Idempotent.
+    pub(super) async fn remove_cells_for_stopped_apps(&mut self) -> ConductorResult<()> {
+        let state = self.get_state().await?;
+        let stopped_cells: HashSet<CellId> = state
+            .stopped_apps()
+            .flat_map(|(_, app)| app.all_cells().cloned().collect::<HashSet<_>>())
+            .collect();
+        for (_, cell) in self
+            .cells
+            .iter()
+            .filter(|(id, _)| stopped_cells.contains(id))
+        {
+            cell.cell.cleanup().await?;
```

Comment:
is this cleanup likely to take time? perhaps we should return this for processing after the lock is released?

Reply:
can probably ignore this, given this is the state of the other calls in here : )